src/components/PeoplePages/PeoplePage.tsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from 'react';
+import { Person } from '../../types/Person';
+import { getPeople } from '../../api';
+import { PeopleTable } from './PeopleTable';
+import { Loader } from '../Loader';
+
+export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    const loadingPeople = async () => {
+      setIsLoading(true);
+      const dataPeople = await getPeople();
+
+      if (!dataPeople) {
+        setError(true);
+      }
+
+      const peopleWithParents = dataPeople.map(person => {
+        const mother = dataPeople.find((motherPerson) => {
+          return motherPerson.name === person.motherName;
+        });
+        const father = dataPeople.find((fatherPerson) => {
+          return fatherPerson.name === person.fatherName;
+        });
```

Comment:
you can combine logic and find mother and father inside single function

Reply:
you can do something like:
```
  const findParentByName = (parentName: string) => {
    return dataPeople.find(parentPerson => parentPerson.name === parentName);
  };
```
and then use it this way:
   ` const mother = findParentByName(person.motherName);
`